gradle.properties

```diff
@@ -9,32 +9,32 @@ versionSuffix=-SNAPSHOT
 
 # The version of Corda that the shell depends on
 # When creating a release build, this should be changed to point to a specific Corda version (e.g 4.9)
-cordaReleaseVersion=4.12-RC01
+cordaReleaseVersion=4.14-SNAPSHOT
 cordaReleaseGroup=net.corda
-cordaPlatformVersion=141
+cordaPlatformVersion=150
```

Comment:
Platform version for 14 is 160 (I just checked  Corda OS).

Reply:
@adelel1 - updated as suggested.